.github/workflows/ci.yml

```diff
@@ -22,9 +22,9 @@ jobs:
     if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/methylseq') }}"
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         NXF_VER:
-          - "22.10.1"
```

Comment:
Same as fetchngs, we should lock down the version, otherwise we're going to be constantly bumping snapshots.

Oh, actually the software versions could be problematic, we might get too used to bumping things on a Nextflow version bump